tests/main/classic-prepare-image-no-core/task.yaml

```diff
@@ -35,7 +35,7 @@ prepare: |
     echo Running prepare-image
     #shellcheck disable=SC2086
     ARCH="$(dpkg-architecture -qDEB_HOST_ARCH)"
-    su -c "SNAPPY_USE_STAGING_STORE=$SNAPPY_USE_STAGING_STORE snap prepare-image --classic --arch $ARCH --channel $CORE_CHANNEL --snap basic18_*.snap  --snap classic-gadget-18_*.snap $TESTSLIB/assertions/developer1-my-classic-w-gadget-18.model $ROOT"
+    SNAPPY_USE_STAGING_STORE="$SNAPPY_USE_STAGING_STORE" snap prepare-image --classic --arch "$ARCH" --channel "$CORE_CHANNEL" --snap basic18_*.snap  --snap classic-gadget-18_*.snap "$TESTSLIB"/assertions/developer1-my-classic-w-gadget-18.model "$ROOT"
```

Comment:
Silly question, why `SNAPPY_USE_STAGING_STORE="$SNAPPY_USE_STAGING_STORE"`? Does it have any effect?

Reply:
Nice catch, I think this is just a leftover from the "su -c" where this if of courses needed.